src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -32,7 +29,15 @@ export default class SidebarMenu extends React.Component {
     }
   }
   render() {
-    let self = this
+    function includes(array, value, folder) {
```

Comment:
can we move it out of render and rename it to be more app specific. It's not a generic includes anymore.